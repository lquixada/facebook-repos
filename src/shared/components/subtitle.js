import React from 'react'
import styled from 'styled-components'

export const Subtitle = ({children}) => (
  <Wrapper>{children}</Wrapper>
)

const Wrapper = styled.h2`
  color: #3c4146;
  font-family: 'Lato', Helvetica, sans-serif;
  font-size: 3rem;
  font-weight: normal;
  letter-spacing: -.1rem;
  line-height: 1.5;
  vertical-align: middle;
`
