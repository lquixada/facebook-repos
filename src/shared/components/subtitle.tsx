import React from 'react'
import styled from 'styled-components'

interface IProps {
  children: string | undefined
}

export const Subtitle = ({children}: IProps) => (
  <Wrapper>{children}</Wrapper>
)

const Wrapper: any = styled.h2`
  color: #3c4146;
  font-family: 'Lato', Helvetica, sans-serif;
  font-size: 3rem;
  font-weight: normal;
  letter-spacing: -.1rem;
  line-height: 1.5;
  vertical-align: middle;
`
