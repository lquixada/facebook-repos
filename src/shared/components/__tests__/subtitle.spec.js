import React from 'react'
import {mount} from 'enzyme'

import {Subtitle} from '../subtitle'

describe('<Subtitle />', () => {
  it('matches the snapshot', () => {
    const component = mount(<Subtitle>Facebook Repos</Subtitle>)
    expect(component).toMatchSnapshot()
  })
})
