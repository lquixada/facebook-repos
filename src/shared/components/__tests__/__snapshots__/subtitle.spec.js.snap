// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Subtitle /> matches the snapshot 1`] = `
<Subtitle>
  <styled.h2>
    <h2
      className="sc-bdVaJa ketyGn"
    >
      Facebook Repos
    </h2>
  </styled.h2>
</Subtitle>
`;
